server/version/version.go

```diff
@@ -12,14 +12,15 @@ type Server struct{}
 func (s *Server) Version(context.Context, *empty.Empty) (*VersionMessage, error) {
 	vers := argocd.GetVersion()
 	return &VersionMessage{
-		Version:      vers.Version,
-		BuildDate:    vers.BuildDate,
-		GitCommit:    vers.GitCommit,
-		GitTag:       vers.GitTag,
-		GitTreeState: vers.GitTreeState,
-		GoVersion:    vers.GoVersion,
-		Compiler:     vers.Compiler,
-		Platform:     vers.Platform,
+		Version:        vers.Version,
+		BuildDate:      vers.BuildDate,
+		GitCommit:      vers.GitCommit,
+		GitTag:         vers.GitTag,
+		GitTreeState:   vers.GitTreeState,
+		GoVersion:      vers.GoVersion,
+		Compiler:       vers.Compiler,
+		Platform:       vers.Platform,
+		KsonnetVersion: vers.KsonnetVersion,
```

Comment:
This complements my other comment: vers.KsonnetVersion will be unavailable. So `server/version/version.go` will need to import ksutil to run the ksonnet version, to augment the return payload..